src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -11,21 +11,49 @@ $(document).ready(function() {
 function readyInstructorStudentEditPage() {
     
     $(INSTRUCTOR_STUDENT_EDIT_FORM).on('submit', function(event) {
+        
+        var isEmailFieldChanged = $('#newstudentemail').val() !== $('#studentemail').val();
+        var isOpenOrPublishedEmailSentInThisCourse = $('#openorpublishedemailsent').val();
+        
         if ($(INSTRUCTOR_STUDENT_EDIT_FORM).attr('editStatus') === 'mustDeleteResponses') {
             event.preventDefault();
 
             var messageText = 'Editing these fields will result in some existing responses from this student '
                               + 'to be deleted. You may download the data before you make the changes. Are '
                               + 'you sure you want to continue?';
             var okCallback = function() {
-                event.target.submit();
+                
+                if (isEmailFieldChanged && isOpenOrPublishedEmailSentInThisCourse) {
+                    sendEmailToNewEmailOption(event);
+                } else {
+                    event.target.submit();
+                }
             };
 
             BootboxWrapper.showModalConfirmation('Confirm Deletion', messageText, okCallback, null,
                     BootboxWrapper.DEFAULT_OK_TEXT, BootboxWrapper.DEFAULT_CANCEL_TEXT, StatusType.WARNING);
+        } else if (isEmailFieldChanged && isOpenOrPublishedEmailSentInThisCourse) {
+            sendEmailToNewEmailOption(event);
         }
     });
     
+    function sendEmailToNewEmailOption(event) {
+        event.preventDefault();
+        var messageText = 'Do you want to resend past session links of this course to the new email '
+                + $('#newstudentemail').val() + '?';
```

Comment:
`$('#newstudentemail').val()` is called before; it can be a parameter of the function `sendEmailToNewEmailOption` instead.